dokka-integration-tests/gradle/build.gradle.kts

```diff
@@ -1,41 +1,237 @@
 /*
  * Copyright 2014-2023 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license.
  */
-
+import org.gradle.api.tasks.testing.logging.TestExceptionFormat.FULL
+import org.gradle.api.tasks.testing.logging.TestLogEvent.FAILED
+import org.gradle.api.tasks.testing.logging.TestLogEvent.SKIPPED
+import org.gradle.language.base.plugins.LifecycleBasePlugin.VERIFICATION_GROUP
+import org.jetbrains.kotlin.gradle.dsl.ExplicitApiMode.Disabled
 
 plugins {
-    id("dokkabuild.test-integration")
```

Comment:
I've removed this convention for now because there's a split between the Maven/CLI/Gradle subprojects. 

I intend to refactor the Maven & CLI projects to use a similar format to this project. Once that's done it will be easier to extract common configuration into a new convention plugin.

For now I've copied the relevant configuration from the `dokkabuild.test-integration` convention into the Gradle subproject build config.